### PR TITLE
use main dgtal master in appveyor + Contributing file of DGtal

### PR DESCRIPTION
**Pull request by kerautret**

### CODE REVIEW

**Comment by dcoeurjo** [comment]
Contributing missing?

**Comment by kerautret** [comment]
oups ;)

**Comment by kerautret** [comment]
@dcoeurjo can I merge?

**Comment by dcoeurjo** [comment]
Ok I thought you would like to have a specific CONTRIBUTING file for DGtaltools (for instance to require that an entry in the readme is need and to discuss between dgtaltools / dgtaltools-contrib)

**Comment by kerautret** [comment]
indeed, in relation I have also add script to apply new tools (see PR: https://github.com/DGtal-team/DGtalScripts/pull/6) I can specific it

**Comment by kerautret** [comment]
@dcoeurjo that done, with a little more specific with DGtalTools.
(I will then apply a small variant for DGTalTools-contrib)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+The DGtalTools project is mainly defined to host tools with a direct
+link to the [DGtal](http://dgtal.org) library and should exploit some
+specific DGtal features. For instance, the *3dCurvatureViewer* tool is
+based on the [integral invariant
+estimator](http://liris.cnrs.fr/dgtal/doc/nightly/LocalEstimatorsFromSurfel.html).
+
+Contrary to the DGtalTools project, the
+[DGtalTools-contrib](https://github.com/DGtal-team/DGtalTools-contrib)
+project can contain some tools which are not directly related to DGtal
+but which uses some feature of DGtal. Note that the
```
Comment (dcoeurjo) [inline CONTRIBUTING.md]: uses -> use

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+[DGtalTools-contrib](https://github.com/DGtal-team/DGtalTools-contrib)
+project can also contains some development tools or prototype used to
+share recent research in progress.
```
Comment (dcoeurjo) [inline CONTRIBUTING.md]: remove "in progress" ?

**Comment by dcoeurjo** [comment]
merging;)

**Pull request merged**